__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -376,6 +376,7 @@ describe('TESTING ChatSchema', () => {
         members: [],
         targetImage: '',
         targetName: '',
+        reply_data: {},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `reply_data` property seems fine, but it's initialized as an empty object. If there are any required fields in `reply_data`, this might cause issues. Ensure that the tests cover all necessary fields within `reply_data`.
```diff
+        reply_data: {},
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->